media-video/handbrake/handbrake-1.2.2.recipe

```diff
@@ -53,29 +56,37 @@ BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libass$secondaryArchSuffix
 	devel:libavcodec$secondaryArchSuffix
-	devel:libbluray$secondaryArchSuffix >= 2
+	devel:libbluray$secondaryArchSuffix
 	devel:libbz2$secondaryArchSuffix
 	devel:libdvdnav$secondaryArchSuffix
 	devel:libdvdread$secondaryArchSuffix
 	devel:libfreetype$secondaryArchSuffix
 	devel:libiconv$secondaryArchSuffix
 	devel:libjansson$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
 	devel:libmkv$secondaryArchSuffix
 	devel:libmp3lame$secondaryArchSuffix
 	devel:libmp4v2$secondaryArchSuffix
 	devel:libmpeg2$secondaryArchSuffix
 	devel:libogg$secondaryArchSuffix
 	devel:libopus$secondaryArchSuffix
 	devel:libsamplerate$secondaryArchSuffix
+	devel:libspeex$secondaryArchSuffix
 	devel:libtheora$secondaryArchSuffix
 	devel:libvorbis$secondaryArchSuffix
 	devel:libvpx$secondaryArchSuffix
 	devel:libx264$secondaryArchSuffix
+	devel:libx265$secondaryArchSuffix
 	devel:libxml2$secondaryArchSuffix
 	devel:libz$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
+	cmd:autoconf
+	cmd:autoreconf
+	cmd:automake
+	cmd:cmake
```

Comment:
Required?

Reply:
NOTE: Minimize. Cmake does a few autotool checks for this build. Need it to pass checks.